src/frontends/ir/src/ir_deserializer.cpp

```diff
@@ -596,10 +598,21 @@ class MetaDataParser : public ov::Meta {
     }
 
     operator ov::AnyMap&() override {
+        m_accessible_by_pugixml_node = false;
         parse();
         return m_parsed_map;
     }
 
+    const pugi::xml_node& get_pugi_node() const override {
+        if (m_accessible_by_pugixml_node) {
+            m_meta_node = m_meta.child(m_name.c_str());
```

Comment:
```suggestion
            return m_meta.child(m_name.c_str());
```
Can be return directly depends on `m_accessible_by_pugixml_node`? 
Right now on each access we do a copy even if `m_accessible_by_pugixml_node` did not change the state.

Reply:
This is a light copy. pugi::xml_node contains only 1 pointer inside.